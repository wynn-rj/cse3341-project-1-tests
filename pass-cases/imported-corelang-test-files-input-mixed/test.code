program
  int x;
begin
  input x;
  input x;
  input x;
  output x;
end
