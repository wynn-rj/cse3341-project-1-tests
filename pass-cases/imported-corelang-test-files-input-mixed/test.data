3 -25 58
