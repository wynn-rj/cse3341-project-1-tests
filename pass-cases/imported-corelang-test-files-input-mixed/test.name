input mixed (Imported from corelan-test-files)
