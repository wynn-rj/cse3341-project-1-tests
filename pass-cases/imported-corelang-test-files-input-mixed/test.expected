program
  int x;
begin
  input x;
  input x;
  input x;
  output x;
end
58
